lib/SILOptimizer/Differentiation/LinearMapInfo.cpp

```diff
@@ -137,31 +92,58 @@ LinearMapInfo::createBranchingTraceDecl(SILBasicBlock *originalBB,
   // Note: must mark enum as implicit to satisfy assertion in
   // `Parser::parseDeclListDelayed`.
   branchingTraceDecl->setImplicit();
-  // Branching trace enums shall not be resilient.
-  branchingTraceDecl->getAttrs().add(new (astCtx) FrozenAttr(/*implicit*/ true));
   if (genericSig)
     branchingTraceDecl->setGenericSignature(genericSig);
-  computeAccessLevel(branchingTraceDecl, original->getEffectiveSymbolLinkage());
+  switch (original->getEffectiveSymbolLinkage()) {
+  case swift::SILLinkage::Public:
+  case swift::SILLinkage::PublicNonABI:
+    // Branching trace enums shall not be resilient.
+    branchingTraceDecl->getAttrs().add(new (astCtx) FrozenAttr(/*implicit*/ true));
+    branchingTraceDecl->getAttrs().add(new (astCtx) UsableFromInlineAttr(/*Implicit*/ true));
```

Comment:
Can we fix this based on https://github.com/apple/swift/pull/63379#discussion_r1096651096?  Adding `@usableFromInline` is not the right fix, and could just be moving the problem somewhere else, as I think this would eventually fail in TBD verification.

Reply:
Oh, sorry please disregard this. I'm looking at the wrong PR 🤦‍♂️